include/aikido/distance/JointAvoidanceConfigurationRanker.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef AIKIDO_DISTANCE_JOINTAVOIDANCECONFIGURATIONRANKER_HPP_
+#define AIKIDO_DISTANCE_JOINTAVOIDANCECONFIGURATIONRANKER_HPP_
+
+#include "aikido/distance/ConfigurationRanker.hpp"
+
+namespace aikido {
+namespace distance {
+
```

Comment:
Nit: document `JointAvoidanceConfigurationRanker`.

I think we should mention here that only joints with limits contribute to the distance to the lower/upper limits.